pkg/apis/wksprovider/machine/crds/crds_vfsdata.go

```diff
@@ -21,39 +21,39 @@ var CRDs = func() http.FileSystem {
 	fs := vfsgen۰FS{
 		"/": &vfsgen۰DirInfo{
 			name:    "/",
-			modTime: time.Date(2020, 6, 16, 15, 3, 16, 642072692, time.UTC),
+			modTime: time.Date(2019, 8, 29, 13, 22, 55, 802109263, time.UTC),
```

Comment:
More spurious changes

Reply:
This files appears to change every time I call `make`. Should I ignore/exclude this from future commits?

Reply:
I usually fetch the previous one, like `git checkout -- pkg/apis/wksprovider/controller/manifests/manifests_vfsdata.go`, to avoid creating spurious changes (but do check that they aren't real changes to the content).

Another approach would be to keep these changes in a separate commit, so they can be quickly skipped over in review.

Reply:
Good point, reverted for now.